### ISSUE TITLE
Unable to install OpenStreetMapXPlot.jl

### ISSUE THREAD

**Issue by johannspies** [pszufe/OpenStreetMapXPlot.jl#5]
```
> julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.0.3
 _/ |\__'_|_|_|\__'_|  |  Debian ⛬  julia/1.0.3+dfsg-4

julia> using Pkg

julia> Pkg.add(PackageSpec(url="https://github.com/pszufe/OpenStreetMapX.jl"))
  Updating git-repo `https://github.com/pszufe/OpenStreetMapX.jl`
 Resolving package versions...
  Updating `~/.julia/environments/v1.0/Project.toml`
  [5c67c38e] - OpenStreetMapX v0.1.1 #master (https://github.com/pszufe/OpenStreetMapX.jl)
  [86cd37e6] + OpenStreetMapX v0.1.5 #master (https://github.com/pszufe/OpenStreetMapX.jl)
  Updating `~/.julia/environments/v1.0/Manifest.toml`
  [5c67c38e] - OpenStreetMapX v0.1.1 #master (https://github.com/pszufe/OpenStreetMapX.jl)
  [86cd37e6] + OpenStreetMapX v0.1.5 #master (https://github.com/pszufe/OpenStreetMapX.jl)

julia> Pkg.add(PackageSpec(url="https://github.com/pszufe/OpenStreetMapXPlot.jl"))
   Cloning git-repo `https://github.com/pszufe/OpenStreetMapXPlot.jl`
  Updating git-repo `https://github.com/pszufe/OpenStreetMapXPlot.jl`
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package OpenStreetMapX [5c67c38e]:
 OpenStreetMapX [5c67c38e] log:
 ├─OpenStreetMapX [5c67c38e] has no known versions!
 └─restricted to versions * by OpenStreetMapXPlot [d3d4fdd0] — no versions left
   └─OpenStreetMapXPlot [d3d4fdd0] log:
     ├─possible versions are: 0.1.2 or uninstalled
     └─OpenStreetMapXPlot [d3d4fdd0] is fixed to version 0.1.2
Stacktrace:
 [1] check_constraints(::Pkg.GraphType.Graph) at /build/julia-wJr69F/julia-1.0.3+dfsg/usr/share/julia/stdlib/v1.0/Pkg/src/GraphType.jl:935
 [2] Pkg.GraphType.Graph(::Dict{Base.UUID,Set{VersionNumber}}, ::Dict{Base.UUID,Dict{Pkg.Types.VersionRange,Dict{String,Base.UUID}}}, ::Dict{Base.UUID,Dict{Pkg.Types.VersionRange,Dict{String,Pkg.Types.VersionSpec}}}, ::Dict{Base.UUID,String}, ::Dict{Base.UUID,Pkg.Types.VersionSpec}, ::Dict{Base.UUID,Pkg.Types.Fixed}, ::Bool) at /build/julia-wJr69F/julia-1.0.3+dfsg/usr/share/julia/stdlib/v1.0/Pkg/src/GraphType.jl:362
 [3] deps_graph(::Pkg.Types.Context, ::Dict{Base.UUID,String}, ::Dict{Base.UUID,Pkg.Types.VersionSpec}, ::Dict{Base.UUID,Pkg.Types.Fixed}) at /build/julia-wJr69F/julia-1.0.3+dfsg/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:307
 [4] resolve_versions!(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}, ::Nothing) at /build/julia-wJr69F/julia-1.0.3+dfsg/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:371
 [5] resolve_versions! at /build/julia-wJr69F/julia-1.0.3+dfsg/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:316 [inlined]
 [6] #add_or_develop#62(::Array{Base.UUID,1}, ::Symbol, ::Function, ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /build/julia-wJr69F/julia-1.0.3+dfsg/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:1201
 [7] #add_or_develop at ./none:0 [inlined]
 [8] #add_or_develop#13(::Symbol, ::Bool, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /build/julia-wJr69F/julia-1.0.3+dfsg/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:64
 [9] #add_or_develop#12 at ./none:0 [inlined]
 [10] #add_or_develop at ./none:0 [inlined]
 [11] #add_or_develop#10 at /build/julia-wJr69F/julia-1.0.3+dfsg/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:27 [inlined]
 [12] #add_or_develop at ./none:0 [inlined]
 [13] #add#18 at /build/julia-wJr69F/julia-1.0.3+dfsg/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:69 [inlined]
 [14] add(::Pkg.Types.PackageSpec) at /build/julia-wJr69F/julia-1.0.3+dfsg/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:69
 [15] top-level scope at none:0
```
This is on Debian.

**Comment by pszufe** [comment]
I submitted `OpenStreetMapX.jl` for Julia public repository and had to do some version renumbering - most likely this is the reason. 
I have corrected the project files - I am able to run install command on my machine and they also have passed on Travis CI. 
Try to rerun (and let me know if it now works)
`Pkg.add(PackageSpec(url="https://github.com/pszufe/OpenStreetMapXPlot.jl"))`

There are strange things happening when one a Julia repository references an unregistered  one. Hopefully it will be solved once OpenStreetMapX gets accepted in Julia METADATA.
